File: alphabeticShift/alphabeticShift.java
package alphabeticShift;

public class alphabeticShift {

    public static String shift(String str){
        char[] ch = str.toCharArray();  

        for(int i = 0; i < ch.length; i++){
            int index = ch[i];
            index = index + 1;
            if(ch[i] == 'z'){
                index = 97;
            }
            ch[i] = (char)index;
        }

        String str1 = new String(ch);
        return str1;
    }
    public static void main(String[] args) {
        System.out.println(shift("zus"));
    }
}

File: alphabeticShift/alphabeticShift.ts
function alphabeticShift(input : string) : string{
    const alphabet: string[] = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j','k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];
    let inputShifted = input.split('');
    console.log(inputShifted);
    for(let i = 0; i < input.length ; i++){
        let index = 0;
        if(inputShifted[i] !== 'z'){
            index = alphabet.indexOf(inputShifted[i]) + 1;
            console.log(index);
        }

        inputShifted[i] = alphabet[index];
    }
    return inputShifted.join('');
}

console.log(alphabeticShift("hello"));
// console.log(alphabeticShift(""));